charts/artifact-hub/values.schema.json

```diff
@@ -916,9 +1019,24 @@
             "title": "Trivy configuration",
             "type": "object",
             "properties": {
+                "enabled": {
+                    "title": "Use trivy security scanner",
```

Comment:
Maybe `Enable Trivy` or `Enable Trivy security scanner`?

In this case, I think it'd be good to explain in the description field that if the `scanner` is enabled but Trivy isn't (via this chart), the user must deploy Trivy by other means and provide the corresponding url to the `scanner` config via the `trivyURL` field.

Reply:
Good point. I hope the descriptions are more explanatory now.

Reply:
They're great, thanks!